addons/mail/static/src/new/views/chatter.js

```diff
@@ -38,6 +38,7 @@ export class Chatter extends Component {
         this.state = useState({
             activities: [],
             attachments: [],
+            composerAction: "none",
```

Comment:
It cannot work. When the chatter is switched from aside to bottom, the component changes. This should definitely be kept in a global state :) On the other hand, I believe that the naming + the replacement of `false` by `"none"` makes total sense :+1:

Reply:
but it works actually